jupyter_server/gateway/gateway_client.py

```diff
@@ -12,12 +12,13 @@
 from abc import ABC, ABCMeta, abstractmethod
 from datetime import datetime, timezone
 from email.utils import parsedate_to_datetime
-from http.cookies import SimpleCookie
+from http.cookies import Morsel, SimpleCookie
```

Comment:
```suggestion
from http.cookies import SimpleCookie
```
I noticed that the Morsel is already imported below in the `if ty.TYPE_CHECKING` block. From my understanding, these changes still only require Morsel for type checking so this could be removed?